HousingSearchListener/V1/Gateway/PersonApiGateway.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Hackney.Core.Logging;
+using HousingSearchListener.V1.Domain.Person;
+using HousingSearchListener.V1.Infrastructure.Exceptions;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace HousingSearchListener.V1.Gateway
+{
+    public class PersonApiGateway : IPersonApiGateway
+    {
+        private readonly IHttpClientFactory _httpClientFactory;
+        private readonly string _getPersonApiRoute;
+        private readonly string _getPersonApiToken;
+
+        private const string PersonApiUrl = "PersonApiUrl";
+        private const string PersonApiToken = "PersonApiToken";
+        private readonly static JsonSerializerOptions _jsonOptions = CreateJsonOptions();
+
+        public PersonApiGateway(IHttpClientFactory httpClientFactory, IConfiguration configuration)
+        {
+            _httpClientFactory = httpClientFactory;
+            _getPersonApiRoute = configuration.GetValue<string>(PersonApiUrl)?.TrimEnd('/');
+            if (string.IsNullOrEmpty(_getPersonApiRoute) || !Uri.IsWellFormedUriString(_getPersonApiRoute, UriKind.Absolute))
+                throw new ArgumentException($"Configuration does not contain a setting value for the key {PersonApiUrl}.");
+
+            _getPersonApiToken = configuration.GetValue<string>(PersonApiToken);
+            if (string.IsNullOrEmpty(_getPersonApiToken))
+                throw new ArgumentException($"Configuration does not contain a setting value for the key {PersonApiToken}.");
+        }
+
+        private static JsonSerializerOptions CreateJsonOptions()
+        {
+            var options = new JsonSerializerOptions
```

Comment:
JsonSerializerOptions ar ethe same across those gateways. Perhaps move to single file.